war/src/main/less/abstracts/theme.less

```diff
@@ -313,7 +313,34 @@
   --call-to-action-text-color: var(--alert-info-text-color);
 
   // Colors
-  --green: #1c9146;
-  --orange: #fe820a;
-  --red: #df001c;
+  each(@colors, {
+    --light-@{key}: lighten(@value, 20%);
+    --@{key}: @value;
+    --dark-@{key}: darken(@value, 20%);
+  });
+
+  each(@semantics, {
+    --@{key}-color: @value;
+  })
```

Comment:
Seems to cause the linter to fail on Windows as described in [JENKINS-68887](https://issues.jenkins.io/browse/JENKINS-68887). Perhaps there should be a semicolon after the closing parenthesis?